apis/controller/v1alpha1/devworkspaceoperatorconfig_types.go

```diff
@@ -67,6 +68,16 @@ type Proxy struct {
 	NoProxy string `json:"noProxy,omitempty"`
 }
 
+type StorageSizes struct {
```

Comment:
Not sure if this should have been called `DefaultStorageSize` instead? For consistency with `WorkspaceConfig`

Reply:
`StorageSizes` is fine here, it's just the name of the internal struct.